src/includes/class-sesamy-api-endpoint.php

```diff
@@ -175,7 +173,7 @@ public function format_response( $served, $result, $request, $server ) {
 					case 'text/html':
 						header( 'Content-Type: text/html; charset=' . get_option( 'blog_charset' ) );
 						if ( ! empty( $result ) ) {
-							echo $result->data['data'], wp_allowed_protocols();
+							echo wp_kses_post( $result->data['data'], wp_allowed_protocols() );
```

Comment:
I also see in the commits that you removed this @PradeepWeb77 , if I remove it then the response is not formatted and I see the word `Array` at the end of the content.